anaron/02.CSS/CSS_02_Articles.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<HTML>
+
+<HEAD>
+    <meta charset="utf-8">
+    <TITLE> CSS 02 Articles</TITLE>
+    <style>
+        body {
+            padding: 2, 0, 0, 2;
+        }
+
+        nav {
+            width: 15em;
+            background-color: lightgray;
+            float: left;
+        }
+
+        .articles {
+            margin-left: 15em;
+            text-align: left;
+            background-color: gray;
+            overflow: hidden;
+        }
+
+        footer {
+            background-color: lightgray;
+            margin-bottom: auto;
+            clear: both;
+        }
+
+        .articles1 {

```

Comment:
Why here you add two equal classes, but different names?

Reply:
i tried how i can make this homework. and i fixed my mistakes. Please review.